galadriel/cli.py

```diff
@@ -423,6 +425,58 @@ def _build_image(docker_username: str) -> None:
     click.echo("Successfully built Docker image!")
 
 
+def _get_image_layer_hashes(image_name: str) -> List[str]:
+    """Get the layer hashes of the Docker image and parse its output as JSON."""
+    result = subprocess.run(
+        ["docker", "inspect", image_name],
+        check=True,
+        capture_output=True,
+        text=True,
+    )
+    try:
+        inspect_data = json.loads(result.stdout)
+    except json.JSONDecodeError as exc:
+        click.echo(f"Failed to parse docker inspect output as JSON: {exc}", err=True)
+        raise exc
+
+    if isinstance(inspect_data, list) and inspect_data:
+        rootfs = inspect_data[0].get("RootFS", {})
```

Comment:
`IndexError` if `inspect_data` list is empty. Not sure if can happen but would consider checking inside `if`